sources/Adapters/picoTracker/filesystem/picoTrackerFileSystem.cpp

```diff
@@ -3,6 +3,123 @@
 #include "Application/Utils/wildcard.h"
 #include "System/Console/Trace.h"
 
+#include <string>
+
+// Max filename is actually 256 per FAT std
+static const int MAX_FILENAME_LEN = 128;
```

Comment:
Yes, I use `MAX_FILENAME_LEN` for reading the filename for example when we need to actually open the file, though I'm no longer passing more than 24 chars out to the callers of this class, ie, SampleImportDialog as it can't actually display longer filenames as you mention.